arcade/application.py

```diff
@@ -43,7 +44,7 @@
 ]
 
 
-def get_screens():
+def get_screens() -> list:
```

Comment:
Nit: can be `List[Something]`?  I'm not sure what `Something` should be.

Reply:
It needs to be `List` rather than `list` since earlier python versions don't support the latter.

The following seems to work on my system after adding `from pyglet.canvas import Screen` to the imports:
```suggestion
def get_screens() -> List[Screen]:
```